agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml

```diff
@@ -55,6 +55,7 @@ rules:
   - networking.k8s.io
   resources:
   - networkpolicies
+  - ingresses
```

Comment:
This will require a change in our helm chart too, right? cc @achandras

Reply:
yep, that's true. We should also clean-up `ingresses` from the `extensions` api group that is no longer needed

Reply:
https://github.com/draios/sysdig-cloud-scripts/blob/1de3393a0c2396c2c3558c5a3681a687a46bc181/agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml#L68
cc @fremmi

Reply:
@irozzo-1A is there any downside to just leaving the extensions apigroup for anyone on really old kubernetes? I'm mainly thinking of openshift 3 which we still test, which matches k8s v1.11, I believe

Reply:
no downside, but it's not used anywhere even by old agent versions so I think that makes sense to just remove it